modules/cluster-logging-release-notes-5.4.0.adoc

```diff
@@ -0,0 +1,37 @@
+= {logging-title-uc} 5.4
+The following advisories are available for {logging} 5.4:
+//link:https://access.redhat.com/errata/ [{logging-title} Release 5.4]
+
+[id="openshift-logging-5-4-0-bug-fixes"]
+== Bug fixes
+*	Before this update, the cluster-logging-operator utilized cluster scoped roles and bindings to establish permissions for the prometheus service account to scrape metrics. These permissions were only created when deploying the Operator using the console interface but was missing when deploying from the command line. This fixes that issue by making this role and binding namespace scoped.	(link:https://issues.redhat.com/browse/LOG-2286[LOG-2286])
+
+*	Before this update, a prior change to fix dashboard reconciliation introduced a cross namespace ownerref to the resource which kept the resource from being created. With this update, removal of that ownerref resolves the issue. (link:https://issues.redhat.com/browse/LOG-2163[LOG-2163])
+
+*	Before this update, changes to the metrics dashboards did not deploy because the cluster-logging-operator did not correctly compare existing and desired configmaps containing the dashboard. With this update, the addition of unique hash value to object labels resolves the issue.	(link:https://issues.redhat.com/browse/LOG-2071[LOG-2071])
+
+*	Before this update, the OpenShift Logging Dashboard displayed the wrong pod namespace in the table which displays top producing and collected containers over the last 24 hours. With this update, pod namespaces display correctly.	(link:https://issues.redhat.com/browse/LOG-2069[LOG-2069])
+
+*	Before this update, when the ClusterLogForwarder was setup with Elasticsearch OutputDefault, and Elasticsearch outputs did not have structured keys, the output secret with the default secret used to communicate to the default log store. This update corrects the secret used.	(link:https://issues.redhat.com/browse/LOG-2056[LOG-2056])
+
+*	Before this update, the Logging dashboard displayed an empty CPU graph because of a reference to an invalid metric. With this update, the correct data point has been selected, resolving the issue.	(link:https://issues.redhat.com/browse/LOG-2026[LOG-2026])
+
+*	Before this update, the image included builder tools that are unnecessary at runtime. This update removes those tools.	(link:https://issues.redhat.com/browse/LOG-1927[LOG-1927])
+
+*	Before this update, a name change of the deployed collector in the 5.3 release caused the alert 'fluentnodedown' to generate.	(link:https://issues.redhat.com/browse/LOG-1918[LOG-1918])
+
+*	Before this update, the collector was collecting its own logs due to a refactoring of the component name change. This could lead to a potential feed-back loop of the collector processing its own log that might result in memory and log message size issues. This update excludes collector logs from collection, resolving the issue.	(link:https://issues.redhat.com/browse/LOG-1774[LOG-1774])
```

Comment:
```suggestion
*	Before this update, the log collector was collecting its own logs due to a refactoring of the component name change. This could lead to a potential feedback loop of the collector processing its own log that might result in memory and log message size issues. This update resolves the issue by excluding the collector logs from the collection. (link:https://issues.redhat.com/browse/LOG-1774[LOG-1774])
```